tests/tests/swfs/avm2/vector_splice_fixed_bug_compat/Test.as

```diff
@@ -0,0 +1,22 @@
+// compiled with mxmlc
+
+
+var v = new Vector.<int>(5, true);
+try {
+    v.push(5);
+} catch(e) {
+    trace("push exception caught");
+}
+v.splice(0, 3);
+trace("NO exception on splice")
+
```

Comment:
It would probably be good to trace the vector (size/contents) afterwards

Reply:
done